packages/fxa-content-server/tests/server/routes/validate-email-domain.js

```diff
@@ -57,7 +57,19 @@ registerSuite('routes/validate-email-domain', {
           assert.isTrue(res.json.calledOnceWith({ result: 'MX' }));
         });
       },
-
+      'responds with result {result: "skip" when there is a skip record}':()=>{
+        const dns = {
+          ...ERROR_CODES,
+          //Promises always resolved so no need to give a Resolver
+        };
+        const validateEmailDomainRoute = proxyquireWithDns(dns);
```

Comment:
Since execution shouldn't get to the part of the code that uses `dns`, we don't need to `proxyquire`.  You can just `require` it.  However, you do need to pass a config object into it so that the `enabled` flag is set to false.